gm4_ambrosia/data/ambrosia/functions/honey_bottle/consumption_checks/level_2.mcfunction

```diff
@@ -0,0 +1,12 @@
+# @s = player who drank level 2 of a honey bottle
+# run from advancement ambrosia:take_sip/level_2
+
+# mark player
```

Comment:
Why isn't this just doing `gm4_ambrosia:honey_bottle/give_new_bottle/level_1`

Reply:
We need the delay to overwrite the vanilla glass bottle, so this uses a call from `pulse_check` to delay the operation.